finetune/finetune.py

```diff
@@ -20,10 +20,9 @@
 from torch.distributed.fsdp.fully_sharded_data_parallel import FullOptimStateDictConfig, FullStateDictConfig
 
 import sys
-sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
-
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
`append` no longer works?

Reply:
This is because it has both the finetune directory and finetune.py. If we still use `append`, `from finetune.finetune_config import FinetuneConfig` will try to find finetune_config from finetune.py firstly.